src/main/msp/msp_serial.c

```diff
@@ -388,7 +389,10 @@ static int mspSerialEncode(mspPort_t *msp, mspPacket_t *packet, mspVersion_e msp
     }
 
     // Send the frame
-    return mspSerialSendFrame(msp, hdrBuf, hdrLen, sbufPtr(&packet->buf), dataLen, crcBuf, crcLen);
+	// If it fails, resend until success
+    while (ret==0)
```

Comment:
This is not the right solution and probably not the right place to this. Better add a retry count loop with small delay and fail after a few times.

Reply:
ok，ok, let me modify it again！

Reply:
@ligenxxxx Are you "jack lee" at slack?